src/tests/ftest/util/osa_utils.py

```diff
@@ -544,7 +544,7 @@ def ior_thread(self, pool, oclass, test, flags,
         self.run_ior_with_pool(create_pool=False, create_cont=False,
                                fail_on_warning=fail_on_warning,
                                out_queue=self.out_queue)
-        if not self.out_queue.empty():
+        if fail_on_warning is True and not self.out_queue.empty():
```

Comment:
Nit: Is there a reason why you have "if fail_on_warning is True" but "if fail_on_warning" above?
```suggestion
        if fail_on_warning and not self.out_queue.empty():
```

Reply:
Oh. I fixed one place and forgot the other.. I can do it in my next PR changes (need more test cases for disk fail). Is it ok to land this PR and I will take care of this change on my future PR.

Reply:
Done.